<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>22</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(149, 106, 214);
font: 87 22pt &quot;Segoe UI Black&quot;;</string>
    </property>
    <property name="text">
     <string>NGƯỜI DÙNG MỚI</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_luu">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>380</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string>background-color: rgb(255, 255, 215); color: rgb(149, 106, 214); border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>THÊM</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_tenND">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
        border: 3px solid #3498db;  /* Viền xanh dương */
        background-color: #ffffff</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
    </property>
    <property name="text">
     <string>TÊN ĐĂNG NHẬP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
    </property>
    <property name="text">
     <string>TÊN NGƯỜI DÙNG</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_tenDN">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
        border: 3px solid #3498db;  /* Viền xanh dương */
        background-color: #ffffff</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>250</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
    </property>
    <property name="text">
     <string>GHI CHÚ</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_note">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>280</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
        border: 3px solid #3498db;  /* Viền xanh dương */
        background-color: #ffffff</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
    </property>
    <property name="text">
     <string>MẬT KHẨU MỚI</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_newpass">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>180</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
        border: 3px solid #3498db;  /* Viền xanh dương */
        background-color: #ffffff</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbo_role">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
          background-color: white;
          border: 3px solid #3498db;  /* Viền xanh dương */
          border-radius: 10px;
          padding: 5px;
      }
      QComboBox::drop-down {
          border: none;
          background: transparent;
      }
      QComboBox::down-arrow {
          image: url(down-arrow.png); /* Thay bằng icon của bạn */
          width: 14px;
          height: 14px;
      }</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
    </property>
    <property name="text">
     <string>PHÂN QUYỀN</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_maND">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
        border: 3px solid #3498db;  /* Viền xanh dương */
        background-color: #ffffff</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;UTM Bell&quot;;
color: rgb(241, 121, 184);
</string>
    </property>
    <property name="text">
     <string>MÃ NGƯỜI DÙNG</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>594</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
